### PR TITLE
Assign attrs to created script element

### PR DESCRIPTION
**Pull request by eerikson**
Need this feature to allow optional `crossOrigin` on behance. However, might be nice for any user of this package to define arbitrary properties on `<script />`.

### CODE REVIEW

File: loadScript.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
-module.exports = function loadScript (src, cb) {
+module.exports = function loadScript (src, cb, attrs) {
   var doc = document
   var tag = 'script'
   var firstScript
   var el
   el = doc.createElement(tag)
   firstScript = doc.getElementsByTagName(tag)[0]
+  if (attrs) {
```
[the remark this answers is not in the document]
Comment (mmase) [reply loadScript.js]: could you make `attrs = {}` by default and then just simply (with no conditional):

```js
Object.assign(el, attrs)l
```

[the remark this answers is not in the document]
Comment (nemtsov) [reply loadScript.js]: Please keep in mind that we're not using babel here, so let's keep compatibility to IE11.

[the remark this answers is not in the document]
Comment (mmase) [reply loadScript.js]: carry on!

[the remark this answers is not in the document]
Comment (eerikson) [reply loadScript.js]: yeah im trying to keep it old school in this PR.

**Pull request merged**